.changelog/unreleased/api-breaking/provider/2171-permissionless-ICS.md.md

```diff
@@ -0,0 +1,36 @@
+
+- Deprecate the `chain-id` parameter in favour of `consumer-id` for all transactions(TXs) and queries targeting a unique consumer chain. Below is a list highlighting the changes in the CLI commands. All commands assume the prefix `interchain-security-pd tx|q provider`.
+  - **TXs:**
+    | Command                                 | Message                             |
+    |-----------------------------------------|-------------------------------------|
+    | `assign-consensus-key [consumer-id] [consumer-pubkey]`  | [MsgAssignConsensusKey](https://github.com/cosmos/interchain-security/blob/feat/permissionless/proto/interchain_security/ccv/provider/v1/tx.proto#L256)             |
+    | `opt-in [consumer-id] [consumer-pubkey]`               | [MsgOptIn](https://github.com/cosmos/interchain-security/blob/feat/permissionless/proto/interchain_security/ccv/provider/v1/tx.proto#L256)                          |
+    | `opt-out [consumer-id]`                                | [MsgOptOut](https://github.com/cosmos/interchain-security/blob/feat/permissionless/proto/interchain_security/ccv/provider/v1/tx.proto#L256)                         |
+    | `set-consumer-commission-rate [consumer-id] [commission-rate]` | [MsgSetConsumerCommissionRate](https://github.com/cosmos/interchain-security/blob/feat/permissionless/proto/interchain_security/ccv/provider/v1/tx.proto#L295) |
+
+
+  - **Queries:**
+    | Command                                        | gRPC Endpoint                                              |
```

Comment:
Same about the table. Replace with bullet point list. e.g., 

- Brief description of the query. Command: `consumer-genesis [consumer-id]`.  REST:  `/interchain_security/ccv/provider/consumer_genesis/{consumer_id}`.

Reply:
Updated.